antarest/study/storage/rawstudy/raw_study_service.py

```diff
@@ -179,6 +201,11 @@ def copy(
 
         """
         self._check_study_exists(src_meta)
+        additional_data = StudyAdditionalData(
```

Comment:
also do that for VariantStudyService.create_variant_study and VariantStudyService.copy and StudyService.create